src/hyp3_srg/__main__.py

```diff
@@ -1,5 +1,5 @@
 """
-Standford processor plugin for HyP3
+Standford Radar Group (SRG) Processor plugin for HyP3
```

Comment:
May as well use the same language as the `__init__.py` docstring?
```suggestion
HyP3 plugin for Stanford Radar Group (SRG) SAR Processor
```